tembo-operator/src/cloudnativepg/cnpg.rs

```diff
@@ -1055,6 +1059,20 @@ pub async fn reconcile_cnpg(cdb: &CoreDB, ctx: Arc<Context>) -> Result<(), Actio
             return Err(Action::requeue(Duration::from_secs(10)));
         }
     }
+    // Don't exec into the pod if it's not Ready yet
+    match is_pod_ready(&primary_pod_cnpg) {
+        Ok(true) => {
+            // Pod is ready, continuing
+        }
+        Ok(false) => {
+            info!("In reconcile_cnpg: Pod is not ready, requeuing 10 seconds");
+            return Err(Action::requeue(Duration::from_secs(10)));
+        }
+        Err(err) => {
+            error!("Failed to determine if pod is ready: {err}");
+            return Err(Action::requeue(Duration::from_secs(30)));
```

Comment:
Lets change this to 300 seconds.  If there is an error it should requeue after a longer period of time.